app/assets/javascripts/controllers/SearchCtrl.coffee

```diff
@@ -0,0 +1,29 @@
+@bruseApp.controller 'SearchCtrl', ['FileHandler', '$scope', '$http', (FileHandler, $scope, $http) ->
+  $scope.files = []
+
+  # watches the search input field for changes
+  $scope.$watch "search", () ->
+    if($scope.search != "")
+      # send a search request to the server
+      $http.get('/search/'+$scope.search)
+        .then((response) ->
+          if response.data.files.length > 0
```

Comment:
Oops, unfinished bussiness. My idea was to display an *not found* message to the user. But didn't implement it since it require some validation that the user is finished with its search and isn't going to write anymore,